common/config/config.go

```diff
@@ -71,8 +71,9 @@ const (
 	DEFAULT_GAS_LIMIT                       = 20000
 	DEFAULT_GAS_PRICE                       = 500
 
-	DEFAULT_DATA_DIR      = "./Chain"
-	DEFAULT_RESERVED_FILE = "./peers.rsv"
+	DEFAULT_DATA_DIR                         = "./Chain"
+	DEFAULT_RESERVED_FILE                    = "./peers.rsv"
+	DEFAULT_OPCODE_HEIGH_UPDATE_FIRST uint32 = 4062597
```

Comment:
need distinguish `testnet`, `mainnet` and `solo` mode, you can reference `STATE_HASH_HEIGHT_MAINNET`